tests/installation/welcome.pm

```diff
@@ -76,7 +78,12 @@ sub run {
         assert_screen 'inst-betawarning';
         wait_screen_change { send_key 'ret' };
     }
-    assert_screen 'inst-welcome';
+    if (sle_version_at_least('15') && check_var('DISTRI', 'sle') && get_var('UPGRADE')) {
```

Comment:
use `is_sle` instead of `check_var('DISTRI', 'sle')`

Reply:
Sure, is_sle